### PR TITLE
apigee x trial org 2nd revision

### PR DESCRIPTION
**Pull request by yuriylesyuk**
What's changed, or what was fixed?

- validation checks; 
- stability; 
- adding envoy and gclb if org is already provisioned

- [x] I have run all the tests locally and they all pass.
- [x] I have followed the relevant style guide for my changes.

**CC:** @apigee-devrel-reviewers

### CODE REVIEW

**Review by danistrebel — COMMENTED** [review]
Minor spelling and an old file reference found. LGTM otherwise

File: tools/apigee-x-trial-provision/README.md [hunk 1]
```diff
@@ -27,4 +40,50 @@ export ZONE=
 export AX_REGION=
 ```
 
-At this time, this script is intended as reference, not a complete solution.
+```sh
+./bin/apigee-ngsaas-trial-install.sh
```
[the remark this answers is not in the document]
Comment (yuriylesyuk) [reply tools/apigee-x-trial-provision/README.md:44]: changed to Apigee X in this particular context, but Interestingly, it is referred as 5g-saas elsewhere:
```
 --metadata ENDPOINT="$APIGEE_ENDPOINT",startup-script-url=gs://apigee-5g-saas/apigee-envoy-proxy-release/latest/conf/startup-script-envoy.sh --project "$PROJECT"
```

**Review by danistrebel — APPROVED** [review]

**Pull request merged**